app/src/pro/java/com/kabouzeid/gramophone/CrashLogger.java

```diff
@@ -0,0 +1,19 @@
+package com.kabouzeid.gramophone;
+
+import android.content.Context;
+
+import com.crashlytics.android.Crashlytics;
+import com.crashlytics.android.core.CrashlyticsCore;
+
+import io.fabric.sdk.android.Fabric;
+
+public class CrashLogger {
+    public static void init(Context context) {
+        if (App.isProFlavor()) {
```

Comment:
why is this necessary here?

Reply:
I don't remember if this was actually required or not. I'll test to see if I can remove it tonight.

Reply:
Removed it.